gulpfile.babel.js

```diff
@@ -257,9 +257,18 @@ const generateGulpLinuxDistroTask = (prefix, name, arch) => {
       categories: ['AudioVideo', 'Audio'],
     };
 
+    var pkg_arch = ''
+    if (arch === '32') {
+      pkg_arch = 'i386';
+    } else if (prefix === 'rpm') {
+      pkg_arch = 'x86_64';
+    } else {
```

Comment:
Purely because I like being picky and having my own JS style forced onto others :laughing:   I feel this would make more logical sense as 

```js
let pkg_arch = 'i386';
if (arch === '64') {
    pkg_arch = (prefix === 'rpm' ? 'x86_64' : 'amd64');
}
```

Also pay attention to the use of `let` instead of `var`.  It's an ES6 thing :+1: